Sources/Controllers/BasicAudioController.swift

```diff
@@ -25,21 +25,37 @@
 import UIKit
 import AVFoundation
 
+/// The `PlayerState` indicates the current audio controller state
+public enum PlayerState {
+
+    /// The audio controller is currently playing a sound
+    case playing
+
+    /// The audio controller is currently in pause state
+    case pause
+
+    /// The audio controller is not playing any sound and audioPlayer is nil
+    case none
```

Comment:
Perhaps `stopped` is a better name for this, a `none` state doesn't make sense grammatically